bundles/org.openhab.io.homekit/src/main/java/org/openhab/io/homekit/internal/HomekitImpl.java

```diff
@@ -203,4 +206,15 @@ public void allowUnauthenticatedRequests(boolean allow) {
     public List<HomekitAccessory> getAccessories() {
         return new ArrayList<HomekitAccessory>(this.changeListener.getAccessories().values());
     }
+
+    public void clearHomekitPairings() {
```

Comment:
shouldn't this have an override annotation?

Reply:
right. done